src/bittytax/conv/parsers/binance.py

```diff
@@ -691,7 +691,7 @@ def _parse_binance_statements_margin_row(
 ) -> None:
     row_dict = data_row.row_dict
 
-    if row_dict["Operation"] in ("Margin loan", "Isolated Margin Loan"):
+    if row_dict["Operation"] in ("Margin Loan", "Isolated Margin Loan"):
```

Comment:
Thanks so much for raising this PR.

Please can you make a small change so both "Margin loan" and "Margin Loan" are matched, to keep it backwards compatible.

I just checked some old Binance statements examples and they had it as "Margin loan".

Reply:
Resolved in https://github.com/BittyTax/BittyTax/pull/431/commits/0db4bf84135eaf9e9748f2bcc102fbedee98069e